currency_rate_update/services/update_service_HU_MNB.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (c) 2009 CamptoCamp. All rights reserved.
+#    @author Nicolas Bessi
+#
+#    Abstract class to fetch rates from Yahoo Financial
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from .currency_getter_interface import Currency_getter_interface
+from lxml import html
+import requests
+
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class HU_MNB_getter(Currency_getter_interface):
+    """Implementation of Currency_getter_factory interface
+    for the Hungarian National Bank (http://www.mnb.hu)
+    """
+
+    def get_updated_currency(self, currency_array, main_currency,
+                             max_delta_days):
```

Comment:
max_delta_days is unused, even though the last date of refresh is available in the table of currencies, please add support for max_delta_days as well (only currencies fresher than today-max_delta_days should be registered)